scripts/python-skywater-pdk/skywater_pdk/netlistsvg-generate.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# Copyright 2020 The SkyWater PDK Authors.
+#
+# Use of this source code is governed by the Apache 2.0
+# license that can be found in the LICENSE file or at
+# https://www.apache.org/licenses/LICENSE-2.0
+#
+# SPDX-License-Identifier: Apache-2.0
+
+
+import csv
+import json
+import os
+import sys
+
+
+# prerequisities:   netlistsvg
```

Comment:
This should be in the help text for this command.

Reply:
Added docstrings and script help.